cmd/beekeeper/cmd/delete_bee_cluster.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func (c *command) initDeleteBeeCluster() *cobra.Command {
+	const (
+		optionNameClusterName = "cluster-name"
+		// optionNameTimeout        = "timeout"
```

Comment:
Remove if not needed.

Reply:
added TODO 
should stay there as a reminder